components/dash-core-components/src/fragments/Dropdown.react.js

```diff
@@ -115,14 +115,14 @@ const Dropdown = props => {
         [multi]
     );
 
-    const onInputChange = useCallback(
-        search_value => setProps({search_value}),
-        []
-    );
+    const onInputChange = useCallback(search_value => {
+        setProps({search_value});
+        setSearchValue(search_value);
+    }, []);
 
     useEffect(() => {
         if (
-            !searchable &&
+            !searchValue &&
```

Comment:
Makes sense that we should just need the pointer to the prop. I'm also realizing that there should be an additional test case that covers this. I don't think we need to watch the `search_value`, since the only time the hook updates the options is when `search_value` is "". The only time that it is changed is when someone is typing in the dropdown or someone clicks outside of it. I'll look into that.